examples/blaze/src/main/scala/com/example/http4s/blaze/demo/server/service/GitHubService.scala

```diff
@@ -19,7 +19,7 @@ class GitHubService[F[_]: Sync](client: Client[F]) extends Http4sClientDsl[F] {
 
   private val RedirectUri = s"http://localhost:8080/$ApiVersion/login/github"
 
-  final case class AccessTokenResponse(access_token: String)
+  case class AccessTokenResponse(access_token: String)
```

Comment:
I like the final on case classes.  It doesn't matter much here, but it's a good habit.

Reply:
Yeah, me too, but this one doesn't compile if it's final :)